spec/03-plugins/19-acl/01-api_spec.lua

```diff
@@ -51,7 +51,9 @@ describe("Plugin: acl (API)", function()
             }
           })
           local body = assert.res_status(400, res)
-          assert.equal([[{"group":"group is required"}]], body)
+          assert.are.same({
```

Comment:
these JSON encoding changes are being handled in https://github.com/Mashape/kong/pull/2402; it's not necessary here i dont think, and it's also out of scope for this commit